libsrc/atari/getdefdev.s

```diff
@@ -68,16 +69,23 @@ crvec:  jsr     $FFFF           ; will be set to crunch vector
         sta     __defdev
         iny
         lda     (DOSVEC),y
-        sta     __defdev+1
+done:   sta     __defdev+1
 
 ; Return pointer to default device
 
 finish: lda     #<__defdev
         ldx     #>__defdev
         rts
 
+; XDOS version
+
+xdos:   lda     XDEFDEV
+        bne     done
+
         .data
 
+crvec:	.byte	$4C,$FF,$FF     ; will be set to crunch vector
```

Comment:
> No. If it were, we wouldn't need mnemonics and write all our code using .byte ;-))

Hey, c'mon, everybody reading the library soure code knows (or is supposed to know, at least) the meaning of $4C, isn't (s)he?  :-)

But I'll change it....